src/BenchmarksApps/TechEmpower/PlatformBenchmarks/ChunkedBufferWriter.cs

```diff
@@ -0,0 +1,238 @@
+﻿using System;
+using System.Buffers;
+using System.Buffers.Text;
+using System.Diagnostics;
+using System.Numerics;
+using System.Runtime.CompilerServices;
+
+namespace PlatformBenchmarks;
+
+internal sealed class ChunkedBufferWriter<TWriter> : IBufferWriter<byte> where TWriter : IBufferWriter<byte>
+{
+    private const int DefaultChunkSizeHint = 2048;
+    private static readonly StandardFormat DefaultHexFormat = GetHexFormat(DefaultChunkSizeHint);
+    private static readonly byte[] ChunkTerminator = "\r\n"u8.ToArray();
```

Comment:
`ReadOnlySpan<byte> ChunkTerminator => "\r\n"u8;`

Reply:
You forgot the `>` in `=>`

Reply:
Examples https://github.com/dotnet/runtime/blob/main/src/libraries/System.Reflection.MetadataLoadContext/src/System/Reflection/TypeLoading/General/Utf8Constants.cs

Reply:
Ah man no wonder I could never get that to work 🤦‍♂️